extensions/wikia/Recirculation/js/helpers/GoogleMatchHelper.js

```diff
@@ -35,16 +35,14 @@ define('ext.wikia.recirculation.googleMatchHelper', [
 		}
 
 		$ins = $('<ins>', {
-			'class': 'adsbygoogle recirculation-rail',
-			'style': 'display: block;',
+			'class': 'adsbygoogle recirculation-incontent',
 			'data-ad-client': 'ca-pub-4086838842346968',
-			'data-ad-format': 'autorelaxed',
 			'data-ad-slot': '7831067424',
 			'data-color-link': sass['color-links'],
 			'data-color-bg': sass['color-page'],
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this trailing comma. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3ATrailingComma)